### ISSUE TITLE
model

### ISSUE THREAD

**Issue by lewisxiaoxu** [its-mayank/SqueezeAttention-PyTorch#3]
能写个带有sablock的resnet网络吗

**Comment by its-mayank** [comment]
您始终可以，我没有足够的时间来编写单独的书，但是如果您编写某些书，将很乐意与您进行调试。
另外，我不懂中文，所以如果您会说英语，请让我们使用。
@lewisxiaoxu

**Comment by StuOfBupt** [comment]
can you add the fc layer and loss function module? thank you
